prisma/schema.prisma

```diff
@@ -11,24 +11,66 @@ datasource db {
 }
 
 model Category {
-  id    Int    @id @default(autoincrement())
-  name  String
+  id   Int    @id @default(autoincrement())
+  name String
 }
 
 model News {
-  id    Int    @id @default(autoincrement())
-  title  String
-  content  String
-  createdAt  DateTime   @default(now())
-}
-
-model Profile {
-  id             Int            @id @default(autoincrement())
-  email          String         @unique
-  privilegeLevel PrivilegeLevel @default(PROFILE)
+  id        Int      @id @default(autoincrement())
+  title     String
+  content   String
+  createdAt DateTime @default(now())
 }
 
 enum PrivilegeLevel {
   PROFILE
```

Comment:
❓ I don't get this one so maybe someone can explain, why is privilege level one of "profile" or "admin"? I get "admin", but "profile"?? Can we change "profile" to "none"?

Reply:
Yeah can we change this to 'basic' maybe? 'none' also works

Reply:
yup just did!